packages/strapi-design-system/src/Field/FieldInput.js

```diff
@@ -0,0 +1,71 @@
+import React, { forwardRef } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { useField } from './FieldContext';
+
+const Input = styled.input`
+  border: 1px solid ${({ theme, hasError }) => (hasError ? theme.colors.danger600 : theme.colors.neutral200)};
+  border-radius: ${({ theme }) => theme.borderRadius};
+  padding: ${({ theme }) => `${theme.spaces[2]} ${theme.spaces[4]}`};
+  color: ${({ theme }) => theme.colors.neutral800};
+  background: ${({ theme }) => theme.colors.neutral0};
+  font-weight: 400;
+  font-size: ${14 / 16}rem;
+  display: block;
+  width: 100%;
+
+  ::placeholder {
+    color: ${({ theme }) => theme.colors.neutral500};
+  }
+
+  :disabled {
+    color: ${({ theme }) => theme.colors.neutral500};
+    background: ${({ theme }) => theme.colors.neutral150};
+  }
+`;
+
+const InputWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
```

Comment:
What do you think about creating something like a Flex component to avoid those CSS duplication?

Reply:
I'm pretty sure I can fix this with a grid 🤔

Reply:
Done with `Row` 😭